components/SubTrackDetails/SubTrackDetailsExample.jsx

```diff
@@ -0,0 +1,23 @@
+import SubTrackDetails from './SubTrackDetails'
+import React from 'react'
+
+/** Temporarly div Style can be replaced by tool tip componenet **/
+const divStyle = {
+  backgroundColor:'#47474F',
+  padding:'20px',
+  width:'320px'
+}
+
+const challengeDetails = {
+  name : 'First To Finish (F2F)',
```

Comment:
I didnt get you.. this is example class right telling how its to be used.? What is the change you are expecting here